src/OpenApiValidator.php

```diff
@@ -86,16 +96,29 @@ private static function wrapValidationException(\Throwable $exception, string $s
         while (null !== ($exception = $exception->getPrevious())) {
             $message[] = $exception->getMessage();
 
-            if (!$at && $exception instanceof SchemaMismatch && $breadcrumb = $exception->dataBreadCrumb()) {
-                $at = \implode('.', $breadcrumb->buildChain());
+            if (!$at) {
+                $at = self::extractPathFromException($exception);
+            }
+
+            if ($exception instanceof NotEnoughValidSchemas) {
+                foreach ($exception->innerExceptions() as $option => $innerException) {
+                    $innerAt = self::extractPathFromException($innerException);
+
+                    $message[] = sprintf(
+                        '==> Schema %d: %s%s',
+                        ((int)$option) + 1,
+                        $innerException->getMessage(),
+                        $innerAt ? sprintf(' (at %s)', $innerAt) : '',
+                    );
+                }
             }
         }
 
         return new AssertionFailedError(
             \sprintf(
                 'OpenAPI %s error%s:%s%s',
                 $scope,
-                $at !== null
+                !empty($at)
```

Comment:
Is there a specific reason you changed this to an `empty` check instead of checking against `null`?

Reply:
I think it would be better to adjust `extractPathFromException` that it does not return empty strings.